projects/elysia/apiInfo.json

```diff
@@ -1,16 +1,27 @@
 {
+  "uniswap-v3-subgraph": {
+    "method": "POST",
+    "endpoint": "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3",
+    "body": {
+      "query": "{\n    daiPool: pool(id: \"0xbde484db131bd2ae80e44a57f865c1dfebb7e31f\"){\n    totalValueLockedUSD,\n    totalValueLockedToken0,\n    totalValueLockedToken1,\n    liquidity,\n    sqrtPrice,\n    poolDayData(orderBy: date, orderDirection: desc) {\n        date,\n        token1Price,\n    },\n    }\n    ethPool: pool(id: \"0xc311faebe8802f9cfc91284016d1de9537ec66b7\"){\n    totalValueLockedUSD,\n    totalValueLockedToken0,\n    totalValueLockedToken1,\n    liquidity,\n    sqrtPrice,\n    poolDayData(orderBy: date, orderDirection: desc) {\n        date,\n        token1Price,\n    },\n    }\n    stakedDaiPositions: positions(where: {pool: \"0xbde484db131bd2ae80e44a57f865c1dfebb7e31f\", owner: \"0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d\"}){\n    id,\n    liquidity,\n    depositedToken0,\n    depositedToken1,\n    }\n    stakedEthPositions: positions(where: {pool: \"0xc311faebe8802f9cfc91284016d1de9537ec66b7\", owner: \"0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d\"}){\n    id,\n    liquidity,\n    depositedToken0,\n    depositedToken1,\n    }\n  }"
+    }
+  },
```

Comment:
I dont think the edits to this file are required

Reply:
@waynebruce0x I removed!